### PR TITLE
Remove no longer relevant separate article visual tone

### PR DESCRIPTION
**Pull request by robertberry-zz**
Fixes this issue: https://github.com/guardian/frontend/issues/7740

### CODE REVIEW

File: common/app/model/content.scala [hunk 1]
```diff
@@ -60,25 +60,28 @@ class Content protected (val apiContent: ApiContentWithMeta) extends Trail with
...
   }
+
   lazy val hasTonalHeaderIllustration: Boolean = isLetters
-  lazy val showBylinePic: Boolean = {
-    visualTone != Tags.VisualTone.News && visualTone != Tags.VisualTone.Live &&
-      contentType != GuardianContentTypes.ImageContent &&
-      hasLargeContributorImage && contributors.length == 1 && !hasTonalHeaderByline
-  }
+  
+  lazy val showCircularBylinePicAtSide: Boolean =
```
Comment (robertberry) [inline common/app/model/content.scala]: @sammorrisdesign I've tried to make it clearer what's going on here, but I'm not sure if this is how the design team expected things to work out. Basically, the circular byline pic at the side (as opposed to the big cut out in the top) only appears on Features at the moment, so far as I can tell.

Comment (sammorrisdesign) [inline common/app/model/content.scala]: I'll stop roping @Pearson82 into everything. But he worked on the byline pics.

Chris, is it just features where the circular cuts appear?

Comment (Pearson82) [inline common/app/model/content.scala]: Aye, it's only features. It was on all articles with a contrib pic, but editorial requested features only. 

@robertberry @sammorrisdesign

**Comment by janua** [comment]
:+1:

**Pull request merged**